tests/optics/focus-lens.test.tsx

```diff
@@ -10,10 +11,10 @@ const succ = (input: number) => input + 1
 
 it('basic derivation using focus works', async () => {
   const bigAtom = atom({ a: 0 })
-  const aAtom = focusAtom(bigAtom, (optic) => optic.prop('a'))
+  const propA = (optic: O.OpticFor<{ a: number }>) => optic.prop('a')
```

Comment:
@merisbahti what's the convention to name this function in optics-ts?

Reply:
its called a lens

Reply:
What exactly would you name this function instead of `propA`.

Reply:
Oh, I think I would call it a `focus`, or `focusFunction`, or `focusCallback`.